Library/Homebrew/extend/ENV/super.rb

```diff
@@ -331,6 +341,11 @@ def libcxx
     append_to_cccfg "g" if compiler == :clang
   end
 
+  sig { void }
+  def set_debug_symbols
+    append_to_cccfg "D" if OS.mac?
```

Comment:
We could probably drop `if OS.mac?`. Adding `D` here won't hurt on Linux.

Reply:
Will do.

Reply:
Done in 6551b25b07a82758209a75152ac4b27494f81e5c